entity-service/src/main/kotlin/com/egm/stellio/entity/model/PropertyUpdate.kt

```diff
@@ -0,0 +1,22 @@
+package com.egm.stellio.entity.model
+
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_PROPERTY_VALUE
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_UNIT_CODE_PROPERTY
+import java.time.ZonedDateTime
+
+class PropertyUpdate private constructor(
```

Comment:
Not sure this class brings a lot to the table ... what does it bring compared to do this parsing in Property.kt?

Reply:
Indeed, it can be done in Property.kt (a static method):
    
```
companion object {
        fun parseUpdateFragment(updateFragment: Map<String, List<Any>>): Triple<String?, Any?, ZonedDateTime?> {
            return Triple(
                JsonLdUtils.getPropertyValueFromMapAsString(updateFragment, NGSILD_UNIT_CODE_PROPERTY),
                JsonLdUtils.getPropertyValueFromMap(updateFragment, NGSILD_PROPERTY_VALUE),
                JsonLdUtils.getPropertyValueFromMapAsDateTime(updateFragment, JsonLdUtils.NGSILD_OBSERVED_AT_PROPERTY)
            )
        }
    }
```
What do you think ?